jib-core/src/main/resources/DockerfileTemplate

```diff
@@ -4,5 +4,5 @@ COPY libs @@DEPENDENCIES_PATH_ON_IMAGE@@
 COPY resources @@RESOURCES_PATH_ON_IMAGE@@
 COPY classes @@CLASSES_PATH_ON_IMAGE@@
 
-ENTRYPOINT @@ENTRYPOINT@@
+@@EXPOSED_PORTS@@ENTRYPOINT @@ENTRYPOINT@@
```

Comment:
Should leave some new lines in between

Reply:
I did that so there are no stray newlines when it gets replaced.

Reply:
Yea, the stray newlines should be an implementation fix rather than on the template itself or it decreases the readability of the template.

Reply:
Also, a nit, but perhaps EXPOSE_INSTRUCTIONS is a more applicable naming of the template section?